### DISCUSSION TITLE
Possibility to use APRSGateway

### DISCUSSION THREAD

**Discussion by iw9grl** [stefanolande/YSFBMDirect#8] [Ideas]
Hi
I wrote in English for more visibility :) but I'm an Italian OM !!

First of all many thanks for software you deployed , it is very usefull for YSF direct connect to BM !
I want to ask you if there is a possibility ( as it is possible with YSFGateway ) , to use YSFBMDirect in combination with APRSGateway to send automatically position over aprs.fi of stations that use the repeater/hotspot. 
Thank you so much
Best 73 de IW9GRL

**Comment by stefanolande** [comment]
Hi @iw9grl,

thanks for using YSFBMDirect, your feedback is really appreaciated.

I think your suggestion is feasible, I'll add it to my backlog. At the moment I'm in the middle of a rewrite to make the software more robust and easy to extend.

73 Stefano
